openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -529,17 +529,18 @@ class AccountRetirementPartnerReportView(ViewSet):
     parser_classes = (JSONParser,)
     serializer_class = UserRetirementStatusSerializer
 
-    def _get_orgs_for_user(self, user):
+    @staticmethod
+    def _get_orgs_for_user(user):
         """
         Returns a set of orgs that the user has enrollments with
         """
         orgs = set()
         for enrollment in user.courseenrollment_set.all():
-            org = enrollment.course.org
+            org = enrollment.course_id.org
```

Comment:
Unfortunate to have both `course` and `course_id` as properties here. And the code does no checking AFAICT if the two match:
https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/models.py#L1162-L1179
No action to take here - just an aside...

Reply:
Ah-ha! This seems to be what enables this whole pre-registration functionality. I wouldn't have done it like this - but, carry on...